R/database.R

```diff
@@ -66,13 +68,21 @@ get_excluded_samples = function(tool_name = "slms-3"){
 #' @export
 #'
 #' @examples
+#' library(dplyr)
```

Comment:
I wonder what is your opinion on the need to import packages before the function can work. I noticed the need for parallel too, but I thought it is because it is not listed under imports here. But the dplyr is listed and based on the example it is still required for this function to work. Not in this PR, but is there a way to make this automated, so we can only call the `library(GAMBLR)` and the stuff will work?

Reply:
So this is an update related to getting all the examples to work with build_site (from pkgdown). It requires any libraries (besides the main package) called in the example to be explicitly listed. For example, if the example involves "%>%" then dplyr will have to be loaded prior to calling the pipe. Does that make sense?

Maybe your suggestion will work, I can try to just load GAMBLR, and hopefully, all dependencies will be loaded as well. Thanks!